### PR TITLE
Fix go linting problems.

### PR DESCRIPTION
**Pull request by VoyTechnology**
Fix the problems with Go styling, such as serious `underscores_in_variables`, as well as less pressing issues, but still following the Go's Code Review Comments wiki page.

Side effect is that errors are actually tracable and a full error log message is visible.

I am sorry if this feels like an offence to the original authors of the code - I realize that people don't always have time to write perfect code. If this repo is to be approved into the official list, we need to improve the code health (#5)

And another apology to the reviewer for such a long change, but I don't think splitting it up and depending on other PRs would be worth it.

### CODE REVIEW

File: examples/example.tf [hunk 1]
```diff
@@ -3,7 +3,7 @@ resource "virtualbox_vm" "node" {
     count = 2
     name = "${format("node-%02d", count.index+1)}"
     url = "https://vagrantcloud.com/ubuntu/boxes/trusty64/versions/20180206.0.0/providers/virtualbox.box"
-    image = "./virtualbox-ubuntu.box"
+    image = "./virtualbox.box"
```
Comment (VoyTechnology) [inline examples/example.tf]: This change would not be needed after #40

Comment (ringods) [reply examples/example.tf]: @VoyTechnology since #40 is merged, I guess the indicated change should be reverted?

**Comment by VoyTechnology** [comment]
@ringods I have rebased the changes, PTAL

**Comment by VoyTechnology** [comment]
I will squash the commits post-review

**Pull request merged**